build.go

```diff
@@ -60,9 +59,22 @@ func Build(
 
 		previousSum, _ := targetsLayer.Metadata[WorkspaceSHAKey].(string)
 		if checksum != previousSum {
-			configuration, err := parser.Parse(filepath.Join(context.WorkingDir, "buildpack.yml"))
-			if err != nil {
-				return packit.BuildResult{}, err
+			configuration := BuildConfiguration{}
```

Comment:
If you aren't going to initialize the struct with field values, then you should just use a `var` declaration. Its a bit more idiomatic.
```suggestion
			var configuration BuildConfiguration
```